docs/topics/ref_additional-vertx-resources.adoc

```diff
@@ -3,6 +3,7 @@
 
 * link:https://www.reactivemanifesto.org/[The Reactive Manifesto]
 * link:http://vertx.io[{VertX} project]
+* link:https://www.manning.com/books/vertx-in-action[{VertX} in Action]
```

Comment:
I think using `{VertX}` here is wrong, as that (IIRC) expands to "Eclipse Vert.x", but the book is really only called Vert.x in Action: https://www.manning.com/books/vertx-in-action

Reply:
I thought that odd myself but I followed the convention used for all of the links. I will verify that the other links to make sure  they are correct as well.